custom_typings/escodegen.d.ts

```diff
@@ -3,13 +3,9 @@ declare module 'escodegen' {
   interface GenerateOpts {
     comment?: boolean;
     format?: {
-      indent?: {
-        style?: string;
-        base?: number;
-        adjustMultilineComment: boolean;
-      }
+      indent?:
+          {style?: string; base?: number; adjustMultilineComment: boolean;}
```

Comment:
Not afaict. I've reduced the wrapping some, but haven't found any tricks for object literals yet.